packages/uswds-core/src/styles/_notifications.scss

```
@@ -13,11 +13,14 @@
 
 /* prettier-ignore */
 $uswds-notifications:
-    "\A"
-  + "\A --------------------------------------------------------------------"
-  + "\A \2709  USWDS Notifications"
-  + "\A --------------------------------------------------------------------"
-  + "\A No notifications";
+  "\A"
++ "\A --------------------------------------------------------------------"
++ "\A \2709  USWDS Notifications"
++ "\A --------------------------------------------------------------------"
++ "\A 3.0.0"
++ "\A We deprecated the $output-all-utilities setting."
++ "\A Control utility output with $output-all-utilities."
++ "\A See designsystem.digital.com/documentation/settings for more.";
```

Add a deprecation notification

---

We should advise to use the new `$output-these-utilities` setting right?

```suggestion
+ "\A 3.0.0"
+ "\A We deprecated the $output-all-utilities setting."
+ "\A Control utility output with $output-these-utilities."
+ "\A See designsystem.digital.com/documentation/settings for more.";
```